### PR TITLE
Add `RemoteControlConfig` serialization on Android

### PR DESCRIPTION
**Pull request by strangesource**
## Description
`RemoteControlConfig` is not serialized to be accessible in React Native yet.

## Changes
Serialize `RemoteControlConfig` to be accessible from React Native. 
`onConfigureMediaInfo` is left out intentionally as this is a rather advanced feature that is rarely used.

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

File: android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt [hunk 1]
```diff
@@ -96,9 +98,67 @@ class JsonConverter {
...
 
+        /**
+         * Converts an arbitrary [ReadableMap] to a [RemoteControlConfig].
+         *
+         * @param json JS object representing the [RemoteControlConfig].
+         * @return The generated [RemoteControlConfig].
+         */
+        private fun toRemoteControlConfig(json: ReadableMap?): RemoteControlConfig? {
+            if (json == null) return null
+            val defaultRemoteControlConfig = RemoteControlConfig()
```
Comment (matamegger) [inline android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt:117]: I was already thinking of this pattern in the past. I was a bit reluctant to actually do it, but it probably is a good option for the use case we have here.

**Review by matamegger — APPROVED** [review]
👍

File: android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt (hunk 1 above)
Comment (strangesource) [reply android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt:117]: I took the inspiration from one of our discussions earlier. This is the best (only?) pattern to take the default values defined in the native code as well as be prepared for a potential immutable configuration object in the future.

Comment (matamegger) [reply android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt:117]: I agree.

**Pull request merged**